x/mint/module/module.go

```diff
@@ -50,6 +51,7 @@ func (AppModuleBasic) RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
 // RegisterInterfaces registers the module's interface types
 func (b AppModuleBasic) RegisterInterfaces(r cdctypes.InterfaceRegistry) {
 	types.RegisterInterfaces(r)
+	v1beta1.RegisterInterfaces(r)
```

Comment:
do we need to do this for `v2` `v3` and `v4` of the mint module? Maybe not because they don't affect the message server? v4 does update the query server

Reply:
Yes I couldn't find `v3` tx messages, the last `v4` version is already registered and is actually using `v2` messages